tests/module/Cargo.lock

```diff
@@ -0,0 +1,1327 @@
+# This file is automatically @generated by Cargo.
```

Comment:
With a Cargo.lock file we will not take the latest RedisJSON (unless doing a `cargo update` before build)
Seems we can just remove and ignore the lock.